man/gnt-instance.rst

```diff
@@ -1539,20 +1538,15 @@ immediately.
 If ``--ignore-ipolicy`` is given any instance policy violations occurring
 during this operation are ignored.
 
-If ``--hotplug`` is given any disk and NIC modifications will take
-effect without the need of actual reboot. Please note that this feature
-is currently supported only for KVM hypervisor and there are some
-restrictions: a) NIC/Disk hot-remove should work for QEMU versions >= 1.0
-b) instances with chroot or pool/user security model support disk
-hot-add only for QEMU version > 1.7 where add-fd QMP command exists c)
-if hotplug fails (for any reason) a warning is printed but execution is
-continued d) for existing NIC modification interactive verification is
-needed unless ``--force`` option is passed.
-
-If ``--hotplug-if-possible`` is given then ganeti won't abort in case
-hotplug is not supported. It will continue execution and modification
-will take place after reboot. This covers use cases where instances are
-not running or hypervisor is not KVM.
```

Comment:
There's a interesting hint in the old manual: `where instances are not running`. For a down instance this looks like this:
```
root@master:~# gnt-instance modify --disk add:size=1g test.vm
Sun Jan 26 14:23:02 2025  - WARNING: Error while executing backend function: [Errno 2] No such file or directory
Sun Jan 26 14:23:02 2025  - INFO: Modification will take place without hotplugging.
...
```
Everything works, but would it be possible to check if the instance is up before trying hotplug?